versioned_docs/version-2.6/reference-guides/cluster-configuration/rancher-server-configuration/eks-cluster-configuration.md

```diff
@@ -93,9 +95,9 @@ A launch template ID and version can be provided in order to easily configure th
 | Node Volume Size | The launch template must specify an EBS volume with the desired size | Required |
 | SSH Key | A key to be added to the instances to provide SSH access to the nodes | Optional |
 | User Data | Cloud init script in [MIME multi-part format](https://docs.aws.amazon.com/eks/latest/userguide/launch-templates.html#launch-template-user-data) | Optional |
-| Instance Resource Tags | Tag each EC2 instance in the node group | Optional |
+| Instance Resource Tags | Tag each EC2 instance and its volumes in the node group | Optional |
```

Comment:
```suggestion
| Instance Resource Tags | Tag each EC2 instance in the node group | Optional |
```

There's no mention of a 2.6 backport in #736 or the reference issue https://github.com/rancher/eks-operator/issues/205.